odoo/addons/base/views/res_partner_views.xml

```diff
@@ -200,7 +200,7 @@
             <field name="arch" type="xml">
                 <form string="Partners">
                 <div class="alert alert-warning oe_edit_only" role="alert" attrs="{'invisible': [('same_vat_partner', '=', False)]}">
-                  A partner with the same Tax ID already exists (<field name="same_vat_partner" widget="html" class="d-inline-block"/>), are you sure to create a new one?
+                  A partner with the same Tax ID already exists (<field name="same_vat_partner"/>), are you sure to create a new one?
```

Comment:
The `base_vat` module replaces "Tax ID" with "VAT", so I imagine this PR should add an extra `inherit` part [here](https://github.com/odoo/odoo/blob/master/addons/base_vat/views/res_partner_views.xml), right? Could be anchored easily with an inline span e.g. `<span class="o_vat_label">Tax ID</span>`.

Reply:
@odony I used the span but the xpath replaces takes the whole span **+ the text node until the next html node**
![image](https://user-images.githubusercontent.com/44770049/57281641-ae72fa00-70ab-11e9-841a-b29712719ed1.png)

Reply:
@william-andre that's because the lxml nodes carry the text that follows them in their `tail` attribute, and the framework doesn't preserve that through _replace_ operations. That could be seen as a bug, and I will open a task to change that in master (can't change it in stable).

In the meantime there are a few not-so-pretty workarounds:
- Wrap the span in another node, e.g. `<i>` or `<b>` (for brevity), so the tail text belongs to the parent
- Or use a first xpath with `position="after"` to insert the new span after the existing one (which will transfer the tailing text), and a second xpath with `position="replace"` and no content, to remove the original span

Reply:
Done